Makefile

```diff
@@ -24,6 +24,12 @@ PLACEMENT_REPO      ?= https://github.com/openstack-k8s-operators/placement-oper
 PLACEMENT_BRANCH    ?= master
 PLACEMENTAPI        ?= config/samples/placement_v1beta1_placementapi.yaml
 
+# Sir Glancealot
+GLANCE_IMG       ?= quay.io/openstack-k8s-operators/glance-operator-index:latest
+GLANCE_REPO      ?= https://github.com/openstack-k8s-operators/glance-operator.git
+GLANCE_BRANCH    ?= master
+GLANCEAPI        ?= config/samples/glance_v1beta1_glanceapi.yaml
```

Comment:
should add an equivalent for glance for the service image what we introduced with https://github.com/openstack-k8s-operators/install_yamls/pull/13

Reply:
Should we also update the `config/samples` in the respective operator repos to point to `quay.io/tripleowallabycentos9` instead of `quay.io/tripleotraincentos8`?

Reply:
yes I think that probably makes sense